config/jobs/kubernetes/sig-testing/compatibility-versions-e2e.yaml

```diff
@@ -0,0 +1,52 @@
+periodics:
+- interval: 24h
```

Comment:
you may want to start with 3 or 4 a day, I recommend 4, otherwise the roundtrip of 24 hours is horrible

Reply:
Ah good point, I've updated this to be 6h now